.github/workflows/pull-request-queue.yaml

```diff
@@ -14,9 +14,18 @@ jobs:
     name: enqueue
     runs-on: ubuntu-20.04
     steps:
+      - name: Query vault
+        id: vault
+        uses: hashicorp/vault-action@v2.0.1
+        with:
+          url: https://volt.cap.explore.suse.dev/
+          token: ${{ secrets.VAULT_TOKEN }}
+          secrets: |
+            secret/data/kubecf "github.cfcontainerizationbot-labeler-personal-access-token" | GH_BOT_TOKEN
```

Comment:
Since we refer to it by `steps.…` we can set `exportEnv` to `false` too.

Reply:
See commit f3c7c8d8411a3c291ca677ec5be23283aa41c778